scenes/Game.gd

```diff
@@ -95,7 +96,21 @@ Relays the 'line_cleared' signal to any listeners, and triggers the 'customer fe
 func _on_line_cleared(lines_cleared: int) -> void:
 	emit_signal("line_cleared", lines_cleared)
 	yield(get_tree().create_timer(rand_range(0.3, 0.5)), "timeout")
-	$CustomerView/SceneClip/Scene.feed()
-	for i in range(1, lines_cleared):
+	_fatten_up(1.0 / lines_cleared)
+	for i in range(lines_cleared - 1):
 		yield(get_tree().create_timer(rand_range(0.066, 0.4)), "timeout")
-		$CustomerView/SceneClip/Scene.feed()
+		_fatten_up(1.0 / (lines_cleared - i - 1))
+
+func _fatten_up(fatness_pct: float) -> void:
```

Comment:
Add a method-level comment.

Reply:
Added a method level comment, and renamed the method to be more consistent with the 'feed' method.